src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -39,12 +39,14 @@ if NOT [%ERRORLEVEL%]==[0] (
     echo Warning: An error occurred while configuration https port.
   )
 
+REM
 REM Start the self hosted WCF Test Service
-call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe %*
+REM
+call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe
```

Comment:
Sounds good. I just want to understand if the rest of script will ever be executed if the exe is taskkilled. In this case, the cleanup code in this script will be skipped, but that's not a problem because the reason that exe is taskkilled is due to another cleanup call from build.override.targets.